react-native-purchases-ui/src/index.tsx

```diff
@@ -58,19 +63,53 @@ export type PresentPaywallIfNeededParams = PresentPaywallParams & {
 
 export default class RevenueCatUI {
 
+  private static Defaults = {
+    PRESENT_PAYWALL_DISPLAY_CLOSE_BUTTON: true
+  }
+
   /**
    * The result of presenting a paywall. This will be the last situation the user experienced before the paywall closed.
    * @readonly
    * @enum {string}
    */
   public static PAYWALL_RESULT = PAYWALL_RESULT;
 
-  public static presentPaywall(params: PresentPaywallParams = {}): Promise<PAYWALL_RESULT> {
-    return RNPaywalls.presentPaywall(params);
+  /**
+   * Presents a paywall to the user with optional customization.
+   *
+   * This method allows for presenting a specific offering's paywall to the user. The caller
+   * can decide whether to display a close button on the paywall through the `displayCloseButton`
+   * parameter. By default, the close button is displayed.
+   *
+   * @param {PresentPaywallParams} params - The options for presenting the paywall.
+   * @returns {Promise<PAYWALL_RESULT>} A promise that resolves with the result of the paywall presentation.
+   */
+  public static presentPaywall({
+                                 offering,
+                                 displayCloseButton = RevenueCatUI.Defaults.PRESENT_PAYWALL_DISPLAY_CLOSE_BUTTON
+                               }: PresentPaywallParams = {}): Promise<PAYWALL_RESULT> {
+    return RNPaywalls.presentPaywall(offering?.identifier ?? null, displayCloseButton)
   }
 
-  public static presentPaywallIfNeeded(params: PresentPaywallIfNeededParams): Promise<PAYWALL_RESULT> {
-    return RNPaywalls.presentPaywallIfNeeded(params);
+  /**
+   * Presents a paywall to the user if a specific entitlement is not already owned.
+   *
+   * This method evaluates whether the user already owns the specified entitlement.
+   * If the entitlement is not owned, it presents a paywall for the specified offering (if provided), or the
+   * default offering (if no offering is provided), to the user. The paywall will be presented
+   * allowing the user the opportunity to purchase the offering. The caller
+   * can decide whether to display a close button on the paywall through the `displayCloseButton`
+   * parameter. By default, the close button is displayed.
+   *
+   * @param {PresentPaywallIfNeededParams} params - The parameters for presenting the paywall.
+   * @returns {Promise<PAYWALL_RESULT>} A promise that resolves with the result of the paywall presentation.
+   */
+  public static presentPaywallIfNeeded({
```

Comment:
Do we need to update the API testers?

Reply:
There's been some updates in those https://github.com/RevenueCat/react-native-purchases/pull/855/files#diff-3158b79f24adf3644ec0b4166445131d8e91e371c03fed63c8d2c66a14a722f5R25

Are we missing any update?

Reply:
Oh sorry I missed those